.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -8,7 +8,7 @@ Thank you for your contribution! Follow this checklist to help us incorporate yo
  - [ ] Update `CHANGES.md` with noteworthy changes.
  - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).
 
-See the [Contributor Guide](https://beam.apache.org/contribute) for more tips on [how to make review process smoother](https://beam.apache.org/contribute/get-started-contributing/#make-the-reviewers-job-easier).
+See the [Contributor Guide](https://beam.apache.org/contribute) for more tips on [how to make review process smoother](https://beam.apache.orghttps://github.com/apache/beam/blob/master/CONTRIBUTING.md#make-the-reviewers-job-easier).
 
```

Comment:
is this supposed to be `https://github.com/apache/beam/blob/master/CONTRIBUTING.md#make-the-reviewers-job-easier)`?

Reply:
Fixed